spec/gameSpec.js

```diff
@@ -0,0 +1,112 @@
+'use strict';
+
+describe('Bowling Game', function(){
+  var game;
+  beforeEach(function(){
+    game = new Game
+  });
+
+  describe('#score', function() {
+
+    it ('scorecard is zero by default', function() {
+        expect(game.scoreCard()).toEqual(game.frames);
+    });
+
+    it ('returns the first bowl score', function() {
+      expect(game.scoreCard()).toEqual([]);
+      game.addFrame(4);
+      expect(game.scoreCard()).toEqual([4]);
```

Comment:
Good testing from going to nothing to something, I should have done this in my tests. High visibility!